trie/triedb/hashdb/database.go

```diff
@@ -65,6 +65,17 @@ type ChildResolver interface {
 	ForEach(node []byte, onChild func(common.Hash))
 }
 
+// Config contains the settings for database.
+type Config struct {
+	CleanSize int // Maximum memory allowance (in bytes) for caching clean nodes
```

Comment:
This is kind of hard to interpret IMO. CleanCacheSize should be better, especially when it ends up in a TOML config file.

Reply:
should we also rename `DirtySize` to `DirtyCacheSize`?